components/FilterLeadsCard/FilterLeadsCard.tsx

```diff
@@ -0,0 +1,236 @@
+'use-client';
+
+import { LEADS_DATA } from '@/utils/constants/leadslist-constant';
+import { useState } from 'react';
+import { CustomMultiSelect } from '../CustomMultiSelect/custom-multi-select';
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
+import { DateRangePicker } from 'react-dates';
+import { Moment } from 'moment';
+
+interface statusState {
```

Comment:
Move these interfaces into `models` folder. Also rename this interface to `StatusState` instead of `statusState`.